Examples/Io/Root/src/RootTrackStatesWriter.cpp

```diff
@@ -387,20 +398,12 @@ ActsExamples::ProcessCode ActsExamples::RootTrackStatesWriter::writeT(
       m_t_eQOP.push_back(truthQOP);
       m_t_eT.push_back(truthTIME);
 
-      // get the geometry ID
-      auto geoID = surface.geometryId();
-      m_volumeID.push_back(geoID.volume());
-      m_layerID.push_back(geoID.layer());
-      m_moduleID.push_back(geoID.sensitive());
-
-      // get the path length
-      m_pathLength.push_back(state.pathLength());
-
       // expand the local measurements into the full bound space
       Acts::BoundVector meas =
           state.effectiveProjector().transpose() * state.effectiveCalibrated();
       // extract local and global position
       Acts::Vector2 local(meas[Acts::eBoundLoc0], meas[Acts::eBoundLoc1]);
+      // TODO use direction from the track state
```

Comment:
That will be ok for planar surfaces, but don't you have access here to the track state momentum already?